core/src/main/res/anim/transition_left.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Kiwix Android
+  ~ Copyright (c) 2020 Kiwix <android.kiwix.org>
+  ~ This program is free software: you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with this program. If not, see <http://www.gnu.org/licenses/>.
+  ~
+  -->
+
+<set xmlns:android="http://schemas.android.com/apk/res/android">
+  <translate
+    android:duration="@android:integer/config_mediumAnimTime"
+    android:fromXDelta="100%"
```

Comment:
now this one looks wrong, 0 to -100% was correct for this one

Reply:
I tried the current configuration on my phone, and it produces the expected results. That's why I made the change. Should I still revert it?

Reply:
When you swipe the current view should move in the direction of your finger, with this the old page jumps to the right of the screen, moves to where it started and then the new tab flashes up. Are you checking this with unique content on each tab? If you are just using the "new tab" view then the error won't be obvious

Reply:
Yes, I saw that. I missed that earlier. Fixed that too with the latest commit :)